README.md

```diff
@@ -66,15 +66,13 @@ For more detailed information on getting started contributing. We have plenty of
 
 ### Discussion
 
-Say hello on Matrix in the [*Firefox Profiler* channel (*#profiler:mozilla.org*)][matrix].
+Say hello on Matrix in the [*Firefox Profiler* channel][matrix].
```

Comment:
I would prefer to keep this as it was (so that folks can also join manually if they want).

Reply:
Thank u for pointing out